scalahost/sbt/src/sbt-test/sbt-scalahost/semantic-example-fat/build.sbt

```diff
@@ -1,6 +1,9 @@
 scalaVersion in ThisBuild := "2.11.11"
 scalacOptions in ThisBuild += "-deprecation"
 scalacOptions in ThisBuild += "-Ywarn-unused-import"
+compile in Compile <<= (compile in Compile).dependsOn(Def.task {
+  sys.props("scalameta.semanticdb") = "fat"
```

Comment:
The idiomatic way to do this however is to provide a `scalametaSemanticdb := fat/slim/none` settingKey. I imagine that the `none` will be very useful. I can do that change in sbt-scalahost in a followup PR if you like.